utils/build.ps1

```diff
@@ -1223,7 +1223,7 @@ function Build-SPMProject {
     [string] $Src,
     [string] $Bin,
     [hashtable] $Arch,
-    [switch] $Test = $false,
+    [string] $Test = "no", # options: no (swift build), yes (swift test), parallel (swift test --parallel)
```

Comment:
Please create an enumerator for this, don't use a string typed value.

Reply:
Good suggestion. Updated.